### PR TITLE
Test123

### PR DESCRIPTION
**Pull request by Lee1887**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h1>
     </nav>
     <main class="Main" main>
```
Comment (devchristiansimon) [inline index.html]: class names in lower cases and the main> doesn't fit here because it isnt a self closing tag

File: index.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h2>About-Me</h2>
     <section id="MyLife" class="#MyLife"></section>
```
Comment (devchristiansimon) [inline index.html]: class names and id names should be in different names for style and review reasons :D

Comment (devchristiansimon) [reply index.html]: and no # in classnames.

Comment (Lee1887) [reply index.html]: Okay, that should have changed by now, thanks for pointing that out.
With the main, I really need to double-check how I do it.

**Pull request merged**